diff --git a/node_modules/@kickstartds/core/lib/core/index.d.ts b/node_modules/@kickstartds/core/lib/core/index.d.ts
index 18ce0cc..f773b68 100644
--- a/node_modules/@kickstartds/core/lib/core/index.d.ts
+++ b/node_modules/@kickstartds/core/lib/core/index.d.ts
@@ -1,7 +1,7 @@
 export { domLoaded, inBrowser } from './domLoaded.js';
 export { render, html } from './html.js';
-export { uid } from './component/uid.js';
-export { define } from './component/define.js';
-export { Component } from './component/Component.js';
+export { uid } from '../component/uid.js';
+export { define } from '../component/define.js';
+export { Component } from '../component/Component.js';
 export * from "../_shared/renderFunctions-55b9fda5";
 export { events, lazyChildrenLoaded } from './lazysizes.js';
